sim/midas/src/main/cc/simif.cc

```diff
@@ -15,23 +15,19 @@ double diff_secs(midas_time_t end, midas_time_t start) {
 }
 
 simif_t::simif_t() {
-  pass = true;
-  t = 0;
-  fail_t = 0;
   seed = time(NULL); // FIXME: better initail seed?
   SIMULATIONMASTER_0_substruct_create;
   this->master_mmio_addrs = SIMULATIONMASTER_0_substruct;
   LOADMEMWIDGET_0_substruct_create;
   this->loadmem_mmio_addrs = LOADMEMWIDGET_0_substruct;
-  PEEKPOKEBRIDGEMODULE_0_substruct_create;
-  this->defaultiowidget_mmio_addrs = PEEKPOKEBRIDGEMODULE_0_substruct;
   CLOCKBRIDGEMODULE_0_substruct_create;
   this->clock_bridge_mmio_addrs = CLOCKBRIDGEMODULE_0_substruct;
 }
 
-void simif_t::init(int argc, char** argv, bool log) {
+void simif_t::init(int argc, char** argv) {
+  // Do any post-constructor initialization required before requesting MMIO
+  this->host_init(argc, argv);
   while(!read(this->master_mmio_addrs->INIT_DONE));
-  this->log = log;
```

Comment:
How does this `log` boolean get enabled? This seems to be only used in the `simif_peek_poke` classes (at least from the diff)... but its a `private` member and thus this `log` isn't used?